_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -0,0 +1,110 @@
+notable changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Discussion about routed coinswaps:** Chris Belcher [posted][belcher
+  coinswap] a design document for a routed multi-transaction coinswap
+  implementation to the Bitcoin-Dev mailing list as a follow-up to his
+  previous post in May (see [Newsletter #100][news100 coinswap]).
+  Discussion from ZmnSCPxj and Nadav Kohen focused on checking that the
+  protocol was safe both in its use of cryptography and in ensuring that
+  the expected transactions would get confirmed (rather than alternative
+  transactions attempting theft).  In general, it appeared that all
+  major concerns were addressed and that the protocol would provide
+  several advantageous privacy benefits.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.0-beta][lnd 0.11.0-beta] is now released.   This new major
+  version allows accepting [large channels][topic large channels] (by
+  default, this is off) and contains numerous improvements to its
+  backend features that may be of interest to advanced users (see the
+  [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #14582][] and [#19743][Bitcoin Core #19743] wallet: always do partial spends FIXME:jonatack
+
+- [Bitcoin Core #19550][] adds a new `getindexinfo` RPC that lists each
+  optional index that has been enabled, whether it is fully synced to
+  the tip of the block chain, and (in case it hasn't been fully synced)
+  how many blocks have been indexed so far.
+
+- [C-Lightning #3954][] updates both the `fundpsbt` and `utxopsbt` RPCs
+  so that they can each take a `locktime` parameter that specifies the
+  nLockTime of the transaction to create.  The PR notes that this is
+  "required for [dual funding][bolts #524], where the opener sets [the
+  locktime]".
```

Comment:
```suggestion
  locktime]."
```

Reply:
What you propose is indeed the normal American style but a number of programmers have complained to me about "closing" the sentence before "closing" a quote that was "opened" within that sentence.  I did a quick search and it seems maybe other programing documentation writers have adopted the same policy: `https://github.com/MicrosoftDocs/microsoft-style-guide/issues/99`

So maybe this is something we discuss?  A quick grep says we already have many cases of `\."`, so I think this is ok to leave for now.  Does anyone have a strong opinion?  I have a slight preference towards the normal `."` style Jonas recommends.

Reply:
+1 for `."`